src/cursor/cursor.ts

```diff
@@ -69,9 +74,16 @@ export interface CursorPrivate {
 }
 
 export interface CursorOptions extends CoreCursorOptions {
-  topology?: typeof Topology;
+  tailableRetryInterval?: number;
+  explicitlyIgnoreSession?: boolean;
+  cursor?: Cursor;
+  /** The internal topology of the created cursor */
+  topology?: Topology;
```

Comment:
there seem to be duplicates between `CursorOptions` and `CoreCursorOptions`

Reply:
I think I got them all now, I wonder if we could find tooling to detect this.